instructor/common.py

```diff
@@ -195,6 +195,18 @@ def assignments_edit(request, assignment_pk):
 
     return redirect('common_assignment_modify')
 
+
+def check_assignment_owner(func):
+    def check_owner(request):
```

Comment:
Also, you should replace `request` with `*args, **kwargs` and pass it to `func` below instead of `func(request)`